### PR TITLE
sorqa-163

### PR DESCRIPTION
**Pull request by pk-sgent**
image

### CODE REVIEW

File: sormas-e2e-tests/src/test/resources/features/sanity/web/CaseHospitalization.feature [hunk 1]
```diff
@@ -15,4 +15,34 @@ Feature: Case hospitalization tab e2e test cases
...
+    When I create a new case with specific data
+    Then I check the created data is correctly displayed on Edit case page
+    And I navigate to Hospitalization tab in Cases
+    When I set Patient Admitted at the facility as an inpatient as YES
+    Then I save data in Hospitalization
+    Then I check if Place of stay in hospital popup is displayed
+    And I choose Facility in Place of stay in hospital popup in Case Hospitalization as Standard Einrichtung
+    Then I save the data in Place of stay in hospital popup
+    Then From hospitalization tab I click on the Case tab button
+    And I check if place of stay data was updated in the Case edit tab with Standard Einrichtung
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/resources/features/sanity/web/CaseHospitalization.feature:48]: this text is written in german on the international environment?

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/HospitalizationTabPage.java [hunk 2]
```diff
@@ -49,4 +49,10 @@
   public static final By BLUE_ERROR_EXCLAMATION_MARK =
       By.xpath("//span[@class='v-errorindicator v-errorindicator-info']");
   public static final By BLUE_ERROR_EXCLAMATION_MARK_TEXT = By.xpath("//div[@class='gwt-HTML']");
+  public static final By PLACE_OF_STAY_IN_HOSPITAL_POPUP =
+      By.xpath("//*[contains(text(),'Place of stay in hospital')]");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/HospitalizationTabPage.java:53]: cannot be taken by other identifier? to make it reusable also for DE if needed

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/EditCasePage.java [hunk 3]
```diff
@@ -178,4 +178,8 @@
   public static final By CREATE_NEW_PERSON_CHECKBOX =
       By.xpath("//label[text()='Create a new person']");
   public static final By CREATE_NEW_CASE_CHECKBOX = By.xpath("//label[text()='Create a new case']");
+  public static final By CURRENT_HOSPITALIZATION_POPUP =
+      By.xpath("//*[contains(text(),'Current hospitalization')]");
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/EditCasePage.java:182]: cannot be taken by other identifier? to make it reusable also for DE if needed

File: sormas-e2e-tests/src/test/resources/features/sanity/web/CaseHospitalization.feature (hunk 1 above)
Comment (pk-sgent) [reply sormas-e2e-tests/src/test/resources/features/sanity/web/CaseHospitalization.feature:48]: test is written on the international env, but in that field we have from always German label. I reported it but it still exists

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/HospitalizationTabPage.java (hunk 2 above)
Comment (pk-sgent) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/HospitalizationTabPage.java:53]: if needed we can make selector like PLACE_OF_STAY_IN_HOSPITAL_POPUP_DE, i didn't see any german test for that .. yet

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/EditCasePage.java (hunk 3 above)
Comment (pk-sgent) [reply sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/EditCasePage.java:182]: same as above

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**